backend/index.js

```
@@ -1,9 +1,29 @@
+  console.error('\nApplication cannot start with insecure configuration.\n');
+  console.error('Please fix the issues above and restart the application.\n');
+  process.exit(1);
+}
+
+// Validate configuration and provide development warnings
 const { validateConfig } = require('./config/validator');
 validateConfig({ exitOnError: process.env.NODE_ENV === 'production' });
+
+// Provide development warnings for placeholder values (non-blocking)
+if (process.env.NODE_ENV !== 'production') {
+  validateForDevelopment();
```

Biome (2.4.4)

[error] This variable is used before its declaration.

(lint/correctness/noInvalidUseBeforeDeclaration)